libr/main/radare2.c

```diff
@@ -690,6 +690,13 @@ R_API int r_main_radare2(int argc, const char **argv) {
 			pfile = argv[opt.ind] ? strdup (argv[opt.ind]) : NULL;
 		}
 	}
+
+	if (pfile && pfile[0] == '\0') {
```

Comment:
I prefer !*pfile than pfile[0] == 0. Shorter snd concise. Can you change this in both places?